utils/envvars/parser.go

```diff
@@ -72,13 +72,18 @@
 			continue
 		}
 
+		var err error
 		// skip test environment variables that are handled elsewere with a big warning
 		if strings.HasPrefix(k, "PERCONA_TEST_") {
 			warns = append(warns, fmt.Sprintf("environment variable %q IS NOT SUPPORTED and WILL BE REMOVED IN THE FUTURE", k))
+			if k == "PERCONA_TEST_DBAAS" {
+				envSettings.EnableDBaaS, err = strconv.ParseBool(v)
+				if err != nil {
+					err = fmt.Errorf("invalid value %q for environment variable %q", v, k)
```

Comment:
staticcheck is not wrong there. Let's at least log original parse error with `logrus.Debugf`

Reply:
Actually I already fixed that.

Reply:
Not sure where – `strconv.ParseBool`'s result is ignored

Reply:
I appended err to errs

Reply:
As discussed in Slack, we _do_ ignore original `err` there:
```go
_, err = errors.New("1")
if err != nil {
  err = errors.New("2")
  errs = append(errs, err)
}
```
We would loss "1".

However, the original staticcheck error wasn't about that; it is just that reviewdog can't remove own comments after they are resolved. While it might be helpful to log original error, we could skip doing that because required linter does not complain